doc/dev/api/server/Ppx_syntax.wiki

```diff
@@ -0,0 +1,67 @@
+=RPC ppx syntax extension
+
+The ocsigen-start syntax extension allows to write, in one function, both the
+client and server part of your RPC.
+
+
+==Sections
+
+The location of code execution is specified by section annotations. 3 types
+of constructions are available to perform RPC sections:
+
+<<code language="ocaml"|
+[%%rpc
+  let r = ...
```

Comment:
this is not a valid rpc syntax. I see what you want to do here, but you shouldn't write syntactically invalid code anyway.